### DISCUSSION TITLE
Possible to move text from Paragraph to Custom Span Node?

### DISCUSSION THREAD

**Discussion by devjk1** [ueberdosis/tiptap#2437] [General Questions] [answered]
When a user enters text, the text is thrown like this `<p>user text</p>`.
How can I change Tiptap behavior to `<p><span>user sentence 1</span><span>user sentence 2</span></p>` ?

To clarify, when a user enters text in the editor, I would like the text to be wrapped in `<span>` tags.  Then on hotkey, a new set of `<span>` tags is used when text is entered.

Any help or direction is appreciated.

**Comment by philippkuehn** [comment, marked as answer]
You probably want to create a [Mark](https://tiptap.dev/guide/custom-extensions#create-a-mark) oder inline [Node](https://tiptap.dev/guide/custom-extensions#create-a-node) for that span tag.
